src/test/java/session/SessionTest.java

```diff
@@ -0,0 +1,39 @@
+package session;
+
+import codec.Codec;
+import message.Message;
+import message.MessageType;
+import org.junit.jupiter.api.Test;
+
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+import java.nio.charset.StandardCharsets;
+
+import static org.mockito.Mockito.*;
+
+class SessionTest {
+
+    Codec codec = mock(Codec.class);
+    SocketChannel socketChannel = mock(SocketChannel.class);
+    Session session = new Session(socketChannel, codec);
+    Message tempMessage = new Message(MessageType.MESSAGE, "user", "hello");
```

Comment:
Why those fields do not have any modifiers (private, public, final, static, etc)?